pgcli/pgclirc

```diff
@@ -71,6 +71,18 @@ qualify_columns = if_more_than_one_table
 # When no schema is entered, only suggest objects in search_path
 search_path_filter = False
 
+# Formatting of function arguments in function calls
+# Supported tags: {arg_name}, {arg_num}, {arg_type} and {arg_default}
+# {max_arg_len} is available for for alignment purposes (for multiline arg lists).
+# (See https://docs.python.org/2/library/string.html#formatstrings for details.)
+call_arg_style = {arg_name: <{max_arg_len}} := {arg_default}
+call_arg_display_style = {arg_name}
+# Func calls with > oneliner_max are prefixed with \n\t
+call_arg_oneliner_max = 2
+# Formatting of function arguments in function signatures, e.g. after DROP FUNCTION.
+# Supported tags: {arg_name}, {arg_num}, {arg_type} and {arg_default}
+signature_arg_style = {arg_name} {arg_type}
+
```

Comment:
Can we choose reasonable defaults and remove these config options? 

I'm not a fan of adding more configuration.